io.unobserved.espansogui.json

```diff
@@ -0,0 +1,39 @@
+{
+  "app-id": "io.unobserved.espansoGUI",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable"
+  ],
+  "command": "espanso-gui",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--filesystem=xdg-config/espanso"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
+    "build-args" : [
+      "--share=network"
```

Comment:
build must be offline. This is not allowed.

Reply:
see https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo

Reply:
Sorry, this was left over from my tests and I forgot to remove it. I will do that now.